python/interpret_community/explanation/explanation.py

```diff
@@ -1148,25 +1148,41 @@ def eval_data(self):
         :return: The dataset or dataset ID.
         :rtype: list or str
         """
-        return self._eval_data
+        return self._convert_to_list(self._eval_data)
 
     @property
     def eval_y_predicted(self):
         """Get predicted ys for the evaluation data.
 
         :return: The predicted ys for the evaluation data.
-        :rtype: np.array
+        :rtype: list
         """
-        return self._eval_y_predicted
+        return self._convert_to_list(self._eval_y_predicted)
 
     @property
     def eval_y_predicted_proba(self):
         """Get predicted probability ys for the evaluation data.
 
-        :param eval_ys_predicted_proba: The predicted probability ys for the evaluation data.
-        :type eval_ys_predicted_proba: np.array
+        :return: The predicted probability ys for the evaluation data.
+        :rtype: list
```

Comment:
this is a list[list[float]], is that right?

Reply:
added